craft_parts/executor/environment.py

```diff
@@ -91,7 +91,7 @@ def _basic_environment_for_part(part: Part, *, step_info: StepInfo) -> dict[str,
     :return: A dictionary containing the built-in environment.
     """
     part_environment = _get_step_environment(step_info)
-    paths = [part.part_install_dir, part.stage_dir]
+    paths = [part.part_install_dir, part.stage_dir, part.backstage_dir]
```

Comment:
I would not add this default now, it would be rather dangerous and put us in a situation we cannot back out of. I would reserve it for plugin use only

Reply:
This allows us to link to things in the backstage directory though.